tsconfig.json

```diff
@@ -7,11 +7,13 @@
     "outDir": "./dist/",
     "sourceMap": true,
     "noImplicitAny": true,
+    "noImplicitThis": false,
```

Comment:
I couldn't find a workaround for this one Hapi handler extension function, which uses `this` inside a function with purpose.